browser/ui/views/brave_actions/brave_actions_container.cc

```diff
@@ -240,10 +283,9 @@ views::LabelButton* BraveActionsContainer::GetOverflowReferenceView() const {
 
 // ToolbarActionView::Delegate members
 gfx::Size BraveActionsContainer::GetToolbarActionSize() {
-  // Shields icon should be square, and full-height
-  gfx::Rect rect(gfx::Size(height(), height()));
-  rect.Inset(-GetLayoutInsets(LOCATION_BAR_ICON_INTERIOR_PADDING));
-  return rect.size();
+  // Width > Height to give space for a large bubble (especially for shields).
+  // TODO(petemill): Generate based on toolbar size.
+  return gfx::Size(34, 24);
```

Comment:
The removed code was actually generating a very incorrect size (`height()` here is very small, likely because nothing is making the height of the container fill the height of the toolbar). This explicit size is moved from https://github.com/brave/brave-core/pull/3698/files#diff-469d5e5a21b50831d605cce6cae56765L160